src/backend/expungeservice/models/charge_classifier.py

```diff
@@ -79,11 +80,22 @@ def _crime_against_person(section):
         if section.isdigit() and any(int(section) in statute_range for statute_range in statute_ranges):
             return PersonCrime
 
+
     @staticmethod
     def _traffic_crime(statute):
-        statute_range = range(801, 826)
-        if statute[0:3].isdigit() and int(statute[0:3]) in statute_range:
-            return Level800TrafficCrime
+
+        if statute[:3].isdigit():
+            statute_range = range(801, 826)
+
+            chapter_num = int(statute[:3])
```

Comment:
We should probably extract a local variable here for chapter like

```python
chapter = statute[:3]
```

and replace the two usages.